src/common/applicantDetails/tabs/fragments/FullName.tsx

```diff
@@ -12,6 +12,7 @@ const FullName = () => (
         placeholder="form.private_person.field.first_name.placeholder"
         required
         validate={mustBeNames(5)}
+        readOnly
```

Comment:
The styling for the readOnly field seem a bit incomplete. On first glance you could assume that they are editable. Maybe out of scope for this PR.